website/docs/r/nsxt_security_group.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_security_group"
+sidebar_current: "docs-vcd-resource-nsxt-security-group"
+description: |-
+Provides a resource to manage NSX-T Security Group. Security groups are groups of data center
```

Comment:
Description text should be indented.
<img width="500" alt="Screenshot 2021-05-04 at 14 54 56" src="https://user-images.githubusercontent.com/11331818/117006460-c2788300-ace8-11eb-844f-7fd556cd07a2.png">

Reply:
Shold be better now. My `make website`is broken, but this https://registry.terraform.io/tools/doc-preview looks to be showing good.